py/DREAM/Output/SPIShardPositions.py

```diff
@@ -15,6 +17,70 @@ def __init__(self, name, data, grid, output, attr=list()):
         super().__init__(name=name, data=data, attr=attr, grid=grid, output=output)
         
 
+    def arrivalTime(self):
+        """
+        Estimate the time at which the pellet arrives to the plasma edge.
+        """
+        if 'eq' not in self.grid:
+            raise OutputException("Cannot plot poloidal trajectory when equilibrium data is not stored in output.")
+
+        R0 = self.grid.eq.R0
+        if np.isinf(R0): R0 = 1
+
+        ROverR0 = self.grid.eq.ROverR0_f*R0 - R0
```

Comment:
Is `ROverR0` an accurate name here? Is maybe `RminusR0` better? Or simply `R`, to be consistent with `Z` below? Although, `Z` could be renamed `ZminusZ0` as well